src/AuthorsView.elm

```diff
@@ -146,64 +153,83 @@ affiliationsHeader =
         []
 
 
-renderAffiliation : Int -> ( Affiliation, Int ) -> Html Msg
-renderAffiliation authorId ( affiliation, index ) =
-    div [ class "affiliation form__question-sub-section" ]
-        [ div [ class "form__label" ] [ text ("Affiliation " ++ toString index) ]
-        , div
-            [ class "remove button button--secondary"
-            , onClick (DeleteAffiliation authorId affiliation.id)
-            ]
-            [ text "Remove Affiliation" ]
-        , div
-            [ class "form__question-sub-section--inline" ]
-            [ div [ class "inline-element" ]
-                [ label
-                    [ class "form__label"
-                    , for ("affiliationInstitution-" ++ toString index)
-                    ]
-                    [ text "Institution" ]
-                , input
-                    [ class "form__input institution"
-                    , list "institutions-list"
-                    , name "institution"
-                    , onInput (UpdateInstitution authorId affiliation.id)
-                    , onFocus (SetFocusedIds authorId affiliation.id)
-                    , onKeyDown (SetAffiliationKeyDown affiliation.id)
-                    , value affiliation.institution
+renderAffiliation : Model -> Int -> ( Affiliation, Int ) -> Html Msg
+renderAffiliation model authorId ( affiliation, index ) =
+    let
+        institutionDiv =
+            if model.showInstitution then
+                div [ class "inline-element" ]
+                    [ label
+                        [ class "form__label"
+                        , for ("affiliationInstitution-" ++ toString index)
```

Comment:
I don't think this `for` attribute is being used. Perhaps we want the same id on the input?